README.md

```diff
@@ -1,34 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# WebPack Template
+# JAVASCRIPT CAPSTONE
 
+Main goal of this project was working as a team with gitflow and best practices.
 
+Additionally using two different API's (TvMaze Show API & Microverse Involvement API).
+
+![screenshot](./screenshot.png)
 
 ## Built With
 
+- API
 - WEBPACK
 - HTML
-- CSS
+- SASS
 - JS
 - Bootstrap 5
 
+### Live version
+
+- live: [Javascript-Capstone](https://iwillteixeira.github.io/javascript-capstone/)
+
+### Project Video
+- [Video](https://drive.google.com/file/d/1kI4FmRN8IYxeZeRoA2kIq3AKwvdvPBeB/view)
```

Comment:
- [ ] Let's include in the video a highlight of a piece of code that you guys are proud of.

[OPTIONAL] I recommend including a small presentation of the students participating in the project, aswell as a small presentation of the main goal of the project.